EnglishFilter/sections/specific.txt

```diff
@@ -307,6 +307,9 @@ get-bitcoin.net##div[id^="adm-container-"]
 ||forum.lewdweb.net/service_worker.js
 bevru.club##a[href="https://advertise.exe.io"]
 neowin.net##.sidebar-col .ad
+||rollercoin.com/static/img/referral/banners$domain=krypto-trend.de
```

Comment:
`||rollercoin.com/static/img/referral/banners$domain=krypto-trend.de` you can block it without `$domain` or use `$third-party`